README.md

```diff
@@ -6,6 +6,12 @@
 
 ---
 
+## Original Fork Repo
+
+https://github.com/release-drafter/release-drafter
+
+Forked from this Commit Id: 74d04fd
```

Comment:
You may write it like following. 


This is a fork of [release-drafter#74d04fd](https://github.com/release-drafter/release-drafter/commit/74d04fd) 

Difference with original fork
1. Description is fetched and is available in the template for using
2. Categories are derived from PR title instead of Github labels.